escaping space charcter in file path

I recently clone this repo and run npm install and found that one of dependency that is deepinstall.js is not there in [npmjs](https://www.npmjs.com/search?q=deepinstall.js) so dependecies should be updated.

@Alphasaurs Oh! Thanks for reporting this, looks like the issue is due to a space in the folder name. we are directly using the path without escaping space characters and that is causing the error `Cannot find module 'D:\Open'` 

We'll definitely fix this soon, but as a workaround for now you could try renaming your folder `Open source development` to something without space? This will unblock you and help you to continue using the module. Meanwhile, we'll fix the issue and update here.

--- a/npm/deepinstall.js
+++ b/npm/deepinstall.js
@@ -25,7 +25,7 @@ async.series([
   function (next) {
     console.log('Running pre-package script');
     var prepackagePath = path.resolve(__dirname, 'pre-package.js'),
-      commandOutput = shell.exec('node ' + prepackagePath);
+      commandOutput = shell.exec(`node "${prepackagePath}"`);
 
     if (commandOutput.code !== 0) {
       console.error('Failed while running pre-package.js, here is the error:');

